### DISCUSSION TITLE
Connecting RS485 with wifi using EW11

### DISCUSSION THREAD

**Discussion by mjpsr11** [tagyoureit/nodejs-poolController#690] [RS485] [answered]
I'm using an EW11 connected to my Pentair EasyTouch.

I've successfully configured the EW11 and ping it's IP address. I thought using this IP in the dashboard controller/RS485 Port as a Port Type of 'Network' would be enough.

The server does indicate `info: Net connect (socat) 0 ready and communicating (with the RS485 IP address)`, but I still see 'Nixie Single Body' as the controller in the dashboard and it is not connected to my EasyTouch panel.

Where did I go wrong?

**Comment by rstrouse** [comment, marked as answer]
That is typically enough.  Double check the ip address and port that you configured in dashPanel.  Once njsPC sees its first packet from the EasyTouch the controller will change to your model and begin interrogating the configuration on the panel.  

Also, if you are not seeing packets, make sure you have data a and b connected properly.  The message above seems to indicate that it found the adapter at the ip/port but there is no RS485 communication on the line.  Please disconnect power and swap the data lines on the EW11.  Do not do this while the EasyTouch is running because the RS485 chips on ET cannot handle a short between A/B or pulling either of them high with the 10v power that is on the bus.  

If you are certain that you have A and B to the correct terminals.  Check the comm settings for the EW11.  These should be set to:
Data Bits: 8
Stop Bits: 1
Parity None:
Flow Control: Half Duplex

The socket settings should be setup with a protocol of TCP-SERVER.

**Reply by mjpsr11** [reply]
It was wires crossed (A/B). Apparently I found a diagram somewhere that showed the connection, but it is wrong. Working now, thanks!

**Reply by tagyoureit** [reply]
And sometimes the different adapters label the A/B or D+/D- backwards.  Glad you figured it out.
